lib/destack/server/utils/index.ts

```diff
@@ -1,4 +1,6 @@
-const formParse = (form, req) => new Promise((resolve, reject) => {
+import fs from 'fs'
+import {NextApiRequest} from 'next'
+const formParse = (form: import("formidable/Formidable"), req:NextApiRequest) => new Promise<Object>((resolve, reject) => {
```

Comment:
Out of curiosity, `import("formidable/Formidable")` is inline:
```js
const formParse = (form: import("formidable/Formidable"), req:NextApiRequest) =&gt; new Promise&lt;Object&gt;((resolve, reject) =&gt; {Write<br class="Apple-interchange-newline">
```

does it not work if imported like so?
```js
import IncomingForm  from 'formidable/Formidable'
```

Reply:
Probably, It was vscode auto import that did it. I'll test it out in a minute.